frontend/javascripts/navbar.tsx

```diff
@@ -859,6 +883,15 @@ function Navbar({
         />,
       );
     }
+    if (isLockedByUser) {
+      trailingNavItems.push(
+        <AnnotationLockedByOwnerTag
```

Comment:
maybe the new property should be called `lockedByOwner`? it's a bit confusing, that the code here has the logic:
```
!isLockedByUser => <LockedByUser />
isLockedByuser => <LockedByOwner />
```